### PR TITLE
Link Champion profiles from entry page & add SEO tags

### PR DESCRIPTION
**Pull request by sophiali23**
## Links:

-   [JIRA Ticket](https://jira.mongodb.org/browse/DEVHUB-785)
-   [Staging Link](https://docs-mongodborg-staging.corp.mongodb.com/CI/devhub/sophia.li/DEVHUB-785/community-champions)

## Description:

-   links champion profiles from the entry page
-   adds tags for SEO and social media on entry and profile pages

## Testing

-   Test that clicking on a champion on the entry page will take you to their profile page and make sure the UI for the hover matches figma design
-   Testing the tags is a bit difficult right now because the ogUrl is pointing to prod, so I get errors when scraping. Using the post inspector on LinkedIn seems to work except the image can't be found. We can test this again when changes are on prod? 

## What types of outside events need to happen for this PR?

-   [ ] Documentation Updates
-   [ ] Product Review
-   [X] Design Review
-   [ ] Release Note Update (only for deployments)

### CODE REVIEW

File: src/pages/community-champions.js [hunk 1]
```diff
@@ -678,12 +683,33 @@ const communityChampionBreadcrumbs = [
     },
 ];
 
+const ogImage = '/public/images/champions-badge.svg';
```
Comment (sophiali23) [inline src/pages/community-champions.js:686]: i get this as the image url, i think it should show up when we scrape the prod link?

Screen Shot 2021-08-11 at 10 59 37 AM

File: src/pages/community-champions.js [hunk 2]
```diff
@@ -678,12 +683,33 @@ const communityChampionBreadcrumbs = [
...
+const ogDescription =
```
Comment (jestapinski) [inline src/pages/community-champions.js:687]: If these are more global, let's declare them in ALL_CAPS please for style!

File: src/pages/community-champions.js [hunk 3]
```diff
@@ -678,12 +683,33 @@ const communityChampionBreadcrumbs = [
...
+    'Champions are a group of passionate, dedicated advocates of the MongoDB community. They keep the community informed and excited about our latest developments and newest offerings. They are the trusted bridge between MongoDB and our community.';
+
 const CommunityChampions = () => {
-    const metadata = useSiteMetadata();
     const useBannerImageWithSpace = useMedia(screenSize.upToSmall);
+    const metadata = useSiteMetadata();
```
Comment (jestapinski) [inline src/pages/community-champions.js:692]:
```suggestion
    const {siteUrl, title} = useSiteMetadata();
```


If we only need a few things, let's destructure this!

File: src/templates/community-champion-profile.js [hunk 4]
```diff
@@ -241,9 +242,24 @@ const CommunityChampionProfile = props => {
     const imageUrl = champion.image ? champion.image.url : null;
     const useMobileLayout = useMedia(screenSize.upToSmallDesktop);
     const metadata = useSiteMetadata();
+    const fullUrl = removePathPrefixFromUrl(`${metadata.siteUrl}${slug}`);
+    const seoTitle = `${fullName} - ${metadata.title}`;
```
Comment (jestapinski) [inline src/templates/community-champion-profile.js:246]:
```suggestion
    const seoTitle = `${fullName} - MongoDB Community Champion`;
```

Nitpick but this feels better to me

**Review by jestapinski — COMMENTED** [review]
Small nitpicks, we are close!

File: src/templates/community-champion-profile.js (hunk 4 above)
Comment (sophiali23) [reply src/templates/community-champion-profile.js]: done! :)

**Review by jestapinski — APPROVED** [review]

**Pull request merged**